src/docs/releases/2.0.0.md

```diff
@@ -4,7 +4,7 @@ Release date: Not yet released
 
 ## Upgrade Instructions
 
-Before upgrading to Orchard Core 2.0, you should first upgrade to 1.8, address any warnings, and only then upgrade to 2.0. Be sure to review the list of breaking changes documented below to ensure a seamless transition.
```

Comment:
Prior to making the leap to Orchard Core 2.0, it's crucial to follow a step-by-step upgrade process. Begin by upgrading your current version to 1.8, resolving any obsolete warnings encountered along the way, and guaranteeing that every site successfully operates on version 1.8.3. This approach ensures that all sites are seamlessly transitioned to 1.8. Subsequently, proceed with confidence to upgrade to 2.0. Prior to initiating this upgrade, thoroughly review the documented list of breaking changes provided below to facilitate a smooth and hassle-free transition.

Reply:
Done, thank you 👍